pkg/common/utils.go

```diff
@@ -62,3 +30,25 @@ func WriteFile(filePath string, data []byte, errToBytes error) error {
 	}
 	return nil
 }
+
+// TODO: use generics
+func RemoveDuplicateValues(stringSlice []string) []string {
+	keys := make(map[string]bool)
+	list := []string{}
+
+	for _, entry := range stringSlice {
+		if _, value := keys[entry]; !value {
+			keys[entry] = true
+			list = append(list, entry)
+		}
+	}
+
+	return list
+}
+
+const tenTwentyFour = 1024
```

Comment:
lol was this to make the linter happy?

I typically see these magic numbers named something more descriptive.
Also there's no reason for the extra memory allocation / math operation of calcuating kilobytes if you only care about megabytes.
```suggestion
const bytesInMB = 1024 * 1024
```

Reply:
Lol yes. Thanks for the code suggestion.